src/main/java/uk/gov/hmcts/reform/document/DocumentMetadataDownloadClientApi.java

```diff
@@ -25,6 +25,7 @@
     Document getDocumentMetadata(
         @RequestHeader(HttpHeaders.AUTHORIZATION) String authorisation,
         @RequestHeader("ServiceAuthorization") String serviceAuth,
+        @RequestHeader("user-roles") String userRoles,
```

Comment:
Those are breaking changes, we are supposed to follow semantic versioning (https://semver.org/)
Also, no tests??